pinot-segment-local/src/test/java/org/apache/pinot/segment/local/upsert/ConcurrentMapPartitionUpsertMetadataManagerTest.java

```diff
@@ -153,12 +157,12 @@ public void testStartFinishOperation() {
   @Test
   public void testAddReplaceRemoveSegment()
       throws IOException {
-    verifyAddReplaceRemoveSegment(HashFunction.NONE, false);
-    verifyAddReplaceRemoveSegment(HashFunction.MD5, false);
-    verifyAddReplaceRemoveSegment(HashFunction.MURMUR3, false);
-    verifyAddReplaceRemoveSegment(HashFunction.NONE, true);
-    verifyAddReplaceRemoveSegment(HashFunction.MD5, true);
-    verifyAddReplaceRemoveSegment(HashFunction.MURMUR3, true);
+    verifyAddReplaceRemoveSegment(NoOpHashFunction.INSTANCE, false);
+    verifyAddReplaceRemoveSegment(MD5UpsertHashFunction.INSTANCE, false);
+    verifyAddReplaceRemoveSegment(Murmur3UpsertHashFunction.INSTANCE, false);
```

Comment:
self-review: forgot to update the test `testHashPrimaryKeys`. We can also remove `HashUtils`.